service/types.go

```diff
@@ -286,3 +287,8 @@ func (m *accountMetadata) UnmarshalJSON(data []byte) error {
 	m.Nonce = nonce
 	return nil
 }
+
+// has0xPrefix validates str begins with '0x' or '0X'.
+func has0xPrefix(str string) bool {
```

Comment:
Please note this comes from go-ethereum: https://github.com/ethereum/go-ethereum/blob/0ba0b81e54483978ab823665afb627ab4999db06/common/hexutil/hexutil.go#L189-L191

Reply:
Don't know if my comment went through last time, but this function is not publicly accessible from go-ethereum.  That's why I reproduced here

Reply:
Yep, not a comment about that we reporduced it. More about attributing.